### ISSUE TITLE
Failed to load after upgrade

### ISSUE THREAD

**Issue by ahloiscreamo** [gnunn1/tilix#961]
Hi,
I am using COPR version for fedora, today tilix failed to load after running daily update command.

terminal output-
```
 root  ~  tilix
object.Exception@../../../.dub/packages/gtk-d-3.6.5/gtk-d/generated/gtkd/gtkd/Loader.d(125): Library load failed (libgdkglext-3.0.so.0): libgdkglext-3.0.so.0: cannot open shared object file: No such file or directory

```

**Comment by Mancooo** [comment]
Hi,

After ugrading Tilix today (on a Fedora 25 system), I have the same problem with the same error showing

**Comment by dsboger** [comment]
I'd say "upgrade again" :). [1.6.1](https://github.com/gnunn1/tilix/releases/tag/1.6.1)

**Comment by ahloiscreamo** [comment]
@dsboger ha i'm going to do that ,thanks man !

**Comment by gnunn1** [comment]
This has been fixed with 1.6.1.
